x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -207,7 +254,7 @@ const NodeEventsInCategoryBreadcrumbs = memo(function ({
           text: (
             <FormattedMessage
               id="xpack.securitySolution.endpoint.resolver.panel.relatedEventList.numberOfEvents"
-              values={{ totalCount: eventCount }}
+              values={{ totalCount: eventCount || '0' }}
```

Comment:
Can you instead hide this text when the event count hasn't loaded yet? Showing `0` during loading could be strange if the request is slow to come back.